private/extensions/maven.bzl

```diff
@@ -292,7 +301,7 @@ def _deduplicate_artifacts_with_root_priority(root_artifacts, non_root_artifacts
             else:
                 warning_parts.append(maven_module)
 
-        print("WARNING: The following maven modules appear in multiple sub-modules with potentially different versions. " +
+        print("WARNING: The following coordinates from `%s` appear in multiple sub-modules with potentially different versions. " % name +
               "Consider adding one of these to your root module to ensure consistent versions:\n\t%s" %
```

Comment:
Should this message be updated to tell people they'll need force_version=True for the coordinates in the root module?